static/scss/product-details.scss

```diff
@@ -0,0 +1,257 @@
+.container {
+  max-width: 1260px;
+  padding: 0 15px;
+  box-sizing: border-box;
+  margin: 0 auto;
+}
+
+.hero-block {
+  font-family: 'Roboto', sans-serif;
```

Comment:
I thought we were moving to Montserrat? (not sure I spelled that right)

Reply:
Ok, I need to remove it from here , I guess, in one of the Arslan's PR it is added in base level.